pkg/glob/glob.go

```diff
@@ -8,7 +8,9 @@ import (
 	"unicode/utf8"
 )
 
-// TODO: Use native path separators instead of always using /.
+// TODO: Use native path separators (i.e., os.PathSeparator) instead of always using `/` when
+// building a path. When parsing a path note that Windows supports both `/` and `\` so Elvish should
+// as well. TBD is whether os.IsPathSeparator handles Windows correctly.
```

Comment:
It does - https://cs.opensource.google/go/go/+/refs/tags/go1.16.4:src/os/path_windows.go;l=12-16